ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml

```diff
@@ -130,12 +130,12 @@ periodics:
         name: release-pull-secret
     serviceAccountName: ci-operator
     volumes:
-    - name: apici-ci-operator-credentials
+    - name: appci-ci-operator-credentials
```

Comment:
Strange.
I cannot use `app.ci-ci-operator-credentials` here. The dot is not allowed in volume name?